pkgs/development/interpreters/python/build-python-package-setuptools.nix

```diff
@@ -13,6 +13,7 @@
 , preShellHook ? ""
 # Execute after shell hook
 , postShellHook ? ""
+, developmentPrefix ? ""
```

Comment:
By reading this variable name I'm not able to infer what it's doing.

Shouldn't the dependencies be packaged in their own derivation instead of running `pip install` in the hook?